modules/standard/Sort.chpl

```diff
@@ -3279,10 +3279,26 @@ record DefaultComparator {
    :returns: 0 if ``a == b``
    :returns: -1 if ``a < b``
    */
+  pragma "last resort"
+  @deprecated("compare with 'a' and 'b' arguments is deprecated, please use compare with 'x' and 'y' arguments instead")
   inline
   proc compare(a, b) {
-    if a < b { return -1; }
-    else if b < a { return 1; }
+    return compare(x=a, y=b);
+  }
+
+    /*
+   Default compare method used in sort functions.
+   Uses the `<` operator to compute the ordering between ``x`` and ``y``.
+   See also `The .compare method`_.
+
+   :returns: 1 if ``y < x``
+   :returns: 0 if ``x == y``
+   :returns: -1 if ``x < y``
+   */
```

Comment:
nit, the indentation on the docstring is weird

Reply:
Lol, I caught that happening on the later one but somehow missed it on this one.  Thanks for the catch!